binderhub/tests/conftest.py

```diff
@@ -48,15 +48,23 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "auth: mark test to run only on auth environments"
     )
-    config.addinivalue_line(
-        "markers", "remote: mark test to run only on remote environments"
-    )
```

Comment:
Should this really be removed? We still have tests marked as remote.

Reply:
Yes, remote was declared twice. This was a duplicate.